src/action-registry/action-registry.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Body, Controller, Get, Param, Patch, Post } from '@nestjs/common';
+import { ActionRegistryService } from './action-registry.service';
+import { CreateActionDto, UpdateActionDto } from './dto';
+
+@Controller('action-registry')
```

Comment:
Let's use just `actions` resource name
```suggestion
@Controller('actions')
```

Reply:
Solved in https://github.com/ggQuest/api-v2-nest/pull/155/commits/95f3a4946b4168316ed5336c418ae51631183201